hls-test-utils/src/Test/Hls.hs

```diff
@@ -126,6 +127,8 @@ runSessionWithServer' plugin conf sconf caps root s = withLock lock $ keepCurren
           }
   x <- runSessionWithHandles inW outR sconf caps root s
   hClose inW
+  touch outW
```

Comment:
does it work? If it stays, please add a comment explaining the rationale for future maintainers

Reply:
unfortunately it doesn't :(
I'll try again putting it before `sleep 0.2`. If it still doesn't work, I think we could ignore this minor issue first -- which seems to occur only on Windows CI, having no effect on test result -- and then cleanup, merge this PR to address #1627